test/test_quantized_nn_mods.py

```diff
@@ -58,6 +58,10 @@ def test_linear_api(self, batch_size, in_features, out_features, use_bias):
         X = torch.rand(batch_size, in_features).float()
         B = torch.rand(out_features).float() if use_bias else None
         qlinear = nnqd.Linear(in_features, out_features)
+        # Run module with default-initialized parameters.
+        # This tests that the constructor is correct.
+        qlinear(X)
```

Comment:
Once this is called, we should check if the initialized weights are of proper dimension. The default constructor path might be different, so the check would be appropriate.

Reply:
resolving -- misunderstood the comment